statistics-service/main.go

```diff
@@ -34,11 +36,10 @@ import (
 // @BasePath /v1
 
 const envVarLogLevel = "LOG_LEVEL"
+const envDisableMigration = "DISABLE_DATA_MIGRATION"
```

Comment:
could you please document somewhere this variable?
I would also add this option to our helm values in the installation charts.